.travis.yml

```diff
@@ -4,4 +4,5 @@ os:
 - linux
 - osx
 env:
-  - OCAML_VERSION=4.03.0 OPAM_VERSION=1.2.1
+  - OCAML_VERSION=4.03.0
```

Comment:
No it was now just unused/useless so I figured it would be more readable to not have conflicting information (we are actually using 1.2.2).